prow/plugins/trigger/generic-comment.go

```diff
@@ -87,6 +95,24 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 		}
 	}
 
+	pr, err := refGetter.PullRequest()
+	if err != nil {
+		return err
+	}
+
+	// Process "help" comments
+	if testHelpRe.MatchString(gc.Body) {
+		available, err := availablePresubmits(c.GitHubClient, gc.Body, org, repo, pr.Base.Ref, pr.Number, presubmits, c.Logger)
+		if err != nil {
+			return err
+		}
+		resp := fmt.Sprintf("No presubmit jobs available for %s/%s", org, repo)
```

Comment:
Should this message include also the branch? We may get here even if jobs are actually available for org/repo, but not the branch for which the PR was submitted.

Also, I think this should be in an `else` rather than conditionally overwritten.

Reply:
Thank you for the review. Updated the message to include the branch, and fixed the condition.